localstack-core/localstack/services/kms/provider.py

```diff
@@ -737,7 +756,7 @@ def _generate_data_key_pair(
             "PublicKey": crypto_key.public_key,
         }
 
-    @handler("GenerateDataKeyPair")
+    @kms_feature.Data
```

Comment:
the `@handler` decorator without any parameter input somehow interfered with the `asf_utils`-tests, I didn't investigate too much and just removed for now. `handlers` with input-parameter are handled correctly.